### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ngdream**
improvment for readme

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
 # How to Contribute
 
-Hello World, all your contributions are welcome!
```
Comment (FanJups) [inline CONTRIBUTING.md:3]: **Hello World .... just fork first.**
We should not delete that, beginners need to understand how it works.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,24 @@
-![Maven Build](https://github.com/osscameroon/js-generator/actions/workflows/maven.yml/badge.svg)
```
Comment (FanJups) [inline README.md:1]: Please, don't remove the badges, it gives some useful information.

**Review by FanJups — COMMENTED** [review]
Thanks for this PR, please I can not merge it now because there some parts we will see together.

File: README.md (hunk 2 above)
Comment (ngdream) [reply README.md:1]: Ok

**Comment by FanJups** [comment]
Please @ngdream , it's better to use another branch than the main one to make a PR. I want to test if the Greetings Action works, please update your main branch with the recent changes I just made on osscameroon main branch.

File: README.md (hunk 2 above)
Comment (FanJups) [reply README.md:1]: I just looked again at your work then I saw that you didn't delete the badges, sorry for the inconvenience. I will do some corrections in order to merge it this weekend, there some good stuff 💯

Comment (FanJups) [reply README.md:1]: Please, keep this style, just add contributors at MIT license

```
![Maven Build](https://github.com/osscameroon/js-generator/actions/workflows/maven.yml/badge.svg)
[![codecov](https://codecov.io/gh/osscameroon/js-generator/branch/main/graph/badge.svg?token=QJEBIRY8JJ)](https://codecov.io/gh/osscameroon/js-generator)
[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

**Comment by FanJups** [comment]
Keep the table of content, great work 💯 , I am a real fan of **Built with love & Built by developers**, we'll keep that. Once everything is ready, we'll merge this week end.

**Comment by FanJups** [comment]
Use "CONTRIBUTING " in capital letters, if not we get a 404 as you can see here, just test by yourself:

https://github.com/ngdream/js-generator/blob/main/CONTRIBUTING.md

https://github.com/ngdream/js-generator/blob/main/contributing.md

`We are happy with every contribution,please have a look to our [contribution guidelines](contributing.md)`

**Pull request merged**